ftw/file/lawgiver.zcml

```diff
@@ -15,6 +15,11 @@
         permissions="ftw.file: Edit advanced fields"
         />
 
+    <lawgiver:map_permissions
+        action_group="manage content settings"
```

Comment:
`map_permissions` accept a comma seperated list of permissions.
The node above also maps to this action group.
I'd therefore move this permission into the node above.

Reply:
Thanks for the hint. I have combined the two action groups.